fractal-puppy/Engine
# Bug/Machete decal not disappearing + UI images Resolution

The following PR contains:
- BUG [#1192](https://app.hacknplan.com/p/85404/kanban?categoryId=0&boardId=265751&taskId=1192&tabId=basicinfo)

**Before:**
If you tried to use machete rain skill while in other skill the machete decal didn't disappear.

**Now:**
The decal doesn't appear while you are using another skill.

- Changed UI textures compression to avoid bad quality.
- Deleted old textures.
- Deleted old Main Menu GOs.

**To test:**
- Use circular attack and try to summon machete rain

**To test 2:**
- Use the Main Menu to see that the quality of the icons is good.
- Use the Inventory and Skills menu in-game to see that the quality of the icons is good.
